gkTally/profitLoss.py

```diff
@@ -21,68 +21,70 @@ def parse_args():
 
 # string containing key words to decide if a transaction is to be considered for
 # profit loss calculation
-profit_loss = ["bought", "sold"]
-investment = ["funding", "wire"]
+profitLossIncludeList = ["bought", "sold"]
+# exclude list - transations that needs to be excluded
+excludeList = ["Bought 100 AAPL"]
```

Comment:
Let us remove all excludeLists. There should only be include LISTS. For now, its ok.